index.d.ts

```diff
@@ -1,8 +1,8 @@
 
-export type Levels = 0 | 1 | 2 | 3 |4;
+export type Levels = 1 | 2 | 3 | 4 | 5;
 
 export const enum levels {
-  DEBUG = 0,
+  DEBUG = 1,
```

Comment:
Is this a breaking change or a fix?

Reply:
Breaking change but releasing under major version 0.